cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -0,0 +1,95 @@
+# HTTP Cache configuration.
+#
+# See cdo-varnish/README.md for more information on the configuration format.
+class HttpCache
+  STATIC_ASSETS = {
```

Comment:
Could clarify in the documentation what happens when there are multiple matching behaviors?  Although it says they are evaluated in top-to-bottom order, it doesn't say that evaluation stops with the first matching path.  In the configuration snippet below, one might expect that the STATIC_ASSETS pattern would strip the cookies after the v3/assets behavior is run.


+        behaviors: [
+          {
+            path: 'v3/assets/*',
+            headers: ALL_HEADERS,
+            cookies: whitelisted_cookies
+          },
+          ...
+          STATIC_ASSETS,